android/java/org/chromium/chrome/browser/brave_news/CardBuilderFeedCard.java

```diff
@@ -282,12 +282,19 @@ public LinearLayout createCard(int type, int position) {
                                 adLogo.setTextColor(R.color.brave_theme_color);
                                 GradientDrawable gd = new GradientDrawable();
                                 gd.setColor(
-                                        mActivity.getResources().getColor(R.color.news_time_color));
+                                        mActivity.getResources().getColor(R.color.news_text_color));
                                 gd.setCornerRadius(15);
                                 gd.setStroke(1, R.color.brave_theme_color);
                                 adLogo.setBackground(gd);
                                 adLogo.setPadding(5, 5, 10, 5);
                                 adLogo.setLayoutParams(adLogoParams);
+                                adLogo.setOnClickListener(new View.OnClickListener() {
+                                    @Override
+                                    public void onClick(View v) {
+                                        openUrlInSameTabAndSavePosition(
+                                                BraveActivity.BRAVE_REWARDS_SETTINGS_URL);
```

Comment:
do we suppose to open rewards settings page on click in that place?

Reply:
yes, it's been discussed in this thread https://bravesoftware.slack.com/archives/CU56GUJ1H/p1641430464002200 . I didn't create an issue for it since it should be part of the specs